packages/api/test/e2e/api/promise-contract.spec.ts

```diff
@@ -6,16 +6,16 @@ import fs from 'fs';
 import path from 'path';
 
 import { Abi } from '@polkadot/api-contract';
-import flipperAbi from '@polkadot/api-contract/test/contracts/flipper.json';
 import testingPairs from '@polkadot/keyring/testingPairs';
 import WsProvider from '@polkadot/rpc-provider/ws';
 import { Address, Hash } from '@polkadot/types';
 
-import { SubmittableResult } from '../../src';
-import ApiPromise from '../../src/promise';
-import describeE2E from '../util/describeE2E';
+import { SubmittableResult } from '../../../src';
+import ApiPromise from '../../../src/promise';
+import describeE2E from '../../util/describeE2E';
 
-const flipperCode = fs.readFileSync(path.join(__dirname, '../../../api-contract/test/contracts/flipper-pruned.wasm')).toString('hex');
+import flipperAbi from '../../util/contracts/flipper.json';
+const flipperCode = fs.readFileSync(path.join(__dirname, '../../util/contracts/flipper-pruned.wasm')).toString('hex');
```

Comment:
Not sure I agree having contract code in util.

Reply:
Ok. I just moved it here because it's only used  here in the e2e tests (apart from one ABI test in api-contract).
Do you generally agree moving the files to the e2e test folder in `api` or would you rather leave them in `api-contract`, even though they are not being used there, but logically belong to that api module?

Reply:
In this case it is a bit of a toss-up -

- it really is contract data
- it actually does get used in the contracts tests (well, the ABIs do - and where not all, they should - just waiting on the new interfaces)

So happy with the e2e tests in api, but the data since it is also used for "normals" (not the wasm, but should be kept with the ABIs)

Reply:
I moved it back now